thoth/common/openshift.py

```diff
@@ -1452,7 +1455,9 @@ def schedule_provenance_checker(
         template_parameters = {
             "THOTH_ADVISER_REQUIREMENTS": requirements,
             "THOTH_ADVISER_REQUIREMENTS_LOCKED": requirements_locked,
-            "THOTH_ADVISER_METADATA": json.dumps({"origin": origin}),
+            "THOTH_ADVISER_METADATA": json.dumps(
+                {"origin": origin, "kebechet_metadata": kebechet_metadata}
```

Comment:
We should sync on this one. I opened https://github.com/thoth-station/adviser/pull/1625 for this and the semantics behind that is:

Keep metadata generic, so anything can be passed in the JSON. Any specific entries like `stack_info` or justification propagated to the adviser output should go to `thoth.adviser` key so we can possibly expand the metadata. WDYT?

Reply:
I'm not sure that I understand what you mean.

Reply:
@fridex Are you saying get rid of https://github.com/thoth-station/common/pull/1103/files#diff-00668a241bbe4ebb4fe69571ac2634fba90bd935bfb267345ec0c946bfe6bdccL1338-L1343 and put it all under the generic argument `metadata` so we can easily add new metadata?